src/schemas/baks/begrunnelse/ba-sak/triggesAv.ts

```diff
@@ -35,4 +34,8 @@ const EØSBegrunnelseTriggere = [
   utdypendeVilkårsvurderingerForEØSTriggere,
 ];
 
-export const triggesAv = [...nasjonaleBegrunnelserTriggere, ...EØSBegrunnelseTriggere];
+export const triggesAv = [
+  ...nasjonaleBegrunnelserTriggere,
+  ...EØSBegrunnelseTriggere,
+  øvrigeTriggere,
```

Comment:
La denne eksplisitt utenfor for å gjøre det tydelig at den gjelder for begge typene begrunnelseTriggere